threading/channels.nim

```diff
@@ -286,19 +288,23 @@ template trySend*[T](c: Chan[T], src: T): bool =
   trySend(c, isolate(src))
 
 proc tryRecv*[T](c: Chan[T], dst: var T): bool {.inline.} =
-  ## Tries to receive a message from the channel `c` and fill `dst` with its value.
-  ## This returns immediately even if no message is found. Doesn't block.
+  ## Tries to receive an item from the channel `c` and fill `dst` with its value.
   ## 
-  ## This can fail for all sort of reasons, including a lack of messages in the channel
-  ## to receive or contention.
+  ## Returns immediately once the lock on the Channel is acquired and an attempt
+  ## at fetching an item from it was made. 
+  ## The proc will not wait for items to appear.
```

Comment:
```suggestion
  ## Doesn't block waiting for the available data in the channel and returns
  ## immediately after an attempt to receive an item was made.
  ## 
  ## .. warning:: Blocking is still possible if another thread uses the blocking
  ##    version of `send`/`recv` and waits for the data/space to appear in the
  ##    channel, thus holding the internal lock to channel's buffer.
```

Reply:
Question about nuances here because I am likely lacking those in my understanding:

Is trySend about waiting for a message to *appear* in the channel's queue, or is it about waiting for a message to become *available* in the channel's queue (implying there are states where it is in the queue, but not available) ?

Reply:
In that case I think we're good with the newest commit. I did change the sentence a bit to split it in 2 and changed the wording marginally:
```txt
  ## Doesn't block waiting for messages in the channel to become available.
  ## Instead returns after an attempt to receive a message was made.
```

I'm mentioning this because the reworded first sentence could imply something incorrect now, so I just want to be on the safe side :smile: